### DISCUSSION TITLE
Should we still use Vault?

### DISCUSSION THREAD

**Discussion by Aging-Developer** [supabase/supabase#23449] [Questions]
As Vault is encrypted with pgsodium (pending deprecation), should we still be using it?

Or is there another free/cheap way of storing a key outside of the DB?  (like storage).

**Comment by GaryAustin1** [comment]
I've heard there will be an "upgrade" so Vault will continue to work, but not clear that is defined yet.

**Reply by Aging-Developer** [reply]
Thanks, Is there any definitive feedback from supabase? i dont want to go live with something and then have to change it with 50,000 app users screaming as the app has stopped working.

**Reply by GaryAustin1** [reply]
I apologize.  That feedback was from a SB employee I asked.  It will be supported, but they have not decided the new method.   If they don't see this here your only other option is support if that is not good enough.
